ts/components/services/NewServiceListItem.tsx

```diff
@@ -136,7 +137,7 @@ export default class NewServiceListItem extends React.PureComponent<
         onLongPress={this.props.onLongPress}
         hideSeparator={this.props.hideSeparator}
         style={styles.listItem}
-        isItemDisabled={!switchValue}
+        isItemDisabled={servicesRedesignEnabled ? false : !switchValue}
```

Comment:
Since we will release in the next few days the feature, what do you think to remove the attribute instead of protecting it with the FF?

Reply:
Since it is a change due to the new services design, I prefer to use always the FF: when the new design is completed released, we'll remove the FF and we'll have a bounce of compile errors indicating where we have to act.

If we leave the props as follows `isItemDisabled={false}` and we remove the FF, it's hard to remember that here we have a "dead code" that eventually could be removed